src/client/client.py

```diff
@@ -13,7 +13,7 @@ class AgentClient:
 
     def __init__(
         self,
-        base_url: str = "http://localhost:80",
+        base_url: str = settings.BASE_URL,
```

Comment:
I’m torn on whether to use the settings in client vs sticking with os for simplicity. I want to add a metadata endpoint so it can get the model names from the server instead of hard coding it.

If we add it to client like this, you’ll need to add pydantic-settings to the top level requirements.txt or this will break the hosted streamlit app

Reply:
I forgot that the client side is separate, because I was experimenting in the same environment. 😅️
In this case, to indeed reduce external dependencies, it's better not to install `pydantic-settings` and directly use `os`.

Reply:
update